.github/workflows/regression-tests.yml

```diff
@@ -115,7 +115,6 @@ jobs:
             lang: vhdl
             python-version: 3.8
             os: ubuntu-latest
-            may_fail: true
```

Comment:
I thought you left this in on purpose because `nightly` IMO makes no claim to be stable and thus would conflate ghdl behavior changes with cocotb PR changes.
But I now realize we have the same setup for icarus master already...

Reply:
Nightly has to pass their regressions. Icarus master does not, but there is no nightly branch there, so we just hope it does break. It hasn't broken to date...

Reply:
Makes sense then.

Reply:
As @ktbarrett said, assets in GHDL are only updated after all tests are successful. Therefore, using nighly assets is as safe as 'stable releases'. For that reason, tagged releases of GHDL do not have assets anymore.